serverless-workflow-examples/serverless-workflow-callback-quarkus/docker-compose/docker-compose.yml

```diff
@@ -83,7 +83,7 @@ services:
 
   data-index:
     container_name: data-index
-    image: quay.io/kiegroup/kogito-data-index-postgresql:${KOGITO_VERSION}
+    image: quay.io/kiegroup/kogito-data-index-postgresql-nightly:latest
```

Comment:
@ricardozanini @nmirasch @rgdoliveira What should we put here? Latest does not make sense as it gets outdated. My suggestion is to pick one concrete version here as we used to do. So maybe `main-2024-02-09` ?

Reply:
I agree @domhanak!